### ISSUE TITLE
"Schematic input does not validate against the Schema" on `ng add`

### ISSUE THREAD

**Issue by erickisos** [Jefiozie/ngx-aws-deploy#287]
**Describe the bug**
I can't add ngx-aws-deploy to my project, every try until now, I get this

```
Skipping installation: Package already installed
? Please select a region: us-east-1
? Please enter the Bucket name: a
? Please enter the subfolder where the files should be uploaded (optional) 
Schematic input does not validate against the Schema: {"region":"us-east-1","bucket":"a","subFolder":""}
Errors:

  Data path "" should have required property '.secretAccessKey'.
```

**To Reproduce**
Just `ng add @jefiozie/ngx-aws-deploy` and it fails

**Expected behavior**
Well, actually I installed the package directly with NPM, but when i call `ng deploy` i get `An unhandled exception occurred: Missing deploy configuration for production`

```
@angular/cli@8.3.24
node v14.2.0
npm 6.14.4
```

**Comment by Jefiozie** [comment]
Hi @erickisos , 

Thanks for reporting this, this is a bug and I just created a PR (#291) for this.
I will publish it as a beta release would you be willing to test it for me if it works after this fix?

 looking forward on your reply.

**Comment by Jefiozie** [comment]
Hi @erickisos,

I've published the beta version [v2.0.1-beta.1](https://www.npmjs.com/package/@jefiozie/ngx-aws-deploy/v/2.0.1-beta.1)

**Comment by erickisos** [comment]
Great!

The `ng add` situation is solved with this version.

**Comment by Jefiozie** [comment]
@erickisos, that is good news 🎉 We will make a official release in the upcoming week.

**Comment by Jefiozie** [comment]
I've just published version `2.0.1` [here](https://www.npmjs.com/package/@jefiozie/ngx-aws-deploy) is the npm link . Thanks for reporting the bug.
